shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -265,9 +334,22 @@ - (NSString*)accessibilityHint {
 - (NSString*)accessibilityValue {
   if (![self isAccessibilityBridgeAlive])
     return nil;
-  if ([self node].value.empty())
-    return nil;
-  return @([self node].value.data());
+
+  if (![self node].value.empty()) {
+    return @([self node].value.data());
+  }
+
+  if ([self node].HasFlag(flutter::SemanticsFlags::kHasToggledState) ||
```

Comment:
Can we remove this code and just continue to use the `UIAccessibilityTraitSelected` as a fallback? (That code should also get a comment explaining why it should never trigger)

Reply:
What was there was wrong for 3 reasons:
 1) 'Selected' has wholly different semantics for users of VoiceOver
 2) The VoiceOver never explained the negative state, only the positive
 3) There was no indication that you could interact with the object

The way things are setup this should never happen, but `{false: ("Autosave", "0", "Button") true: ("Autosave", "1", "Button")}` is more informative and correct than `{false:("Autosave") true:{"Selected", "Autosave"}`

Reply:
Added comments.